libbpf-tools/cpudist.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __CPUDIST_H
+#define __CPUDIST_H
+
+#define MAX_SLOTS    65
```

Comment:
I think 65 is way too much and we'll be wasting memory a lot. I think we can be smarter and use only enough slots to represent reasonable periods of times, with all the (unlikely) longer durations just falling into the last bucket. E.g., if we were to track 8 hours with microsecond precision, we'd need only `log2(8 * 60 * 60 * 1000000) = 35` buckets, so allocating 36 instead of 65 would allow to capture everything within 8 hours precisely + 1 overflow bucket for everything longer than 8 hours. With millisecond precision we are talking about 8000 hours, which is beyond reasonable :)

Reply:
Fixed, thanks a lot.